fern/docs.yml

```diff
@@ -723,6 +716,27 @@ navigation:
           href: https://tradeparadex.github.io/paradex-py/
         - link: CCXT Integration
           href: https://docs.ccxt.com/#/exchanges/paradex
+  - tab: websockets
+    layout:
+      - section: WebSockets Reference
```

Comment:
I know this is how Fern did it initially, but we might want to have this included in the single API Reference tab?

Either that or just rename it to be REST API Ref vs Websocket Ref ?

Reply:
Also just as a note, if we do use the separate tab we need to make sure that we remove the links from docs.yaml

Reply:
Its not 100% the same, but we could also use something like [versioning](https://buildwithfern.com/learn/docs/building-and-customizing-your-docs/versioning) to keep this a bit cleaner in the same tab ;)